docs/blog/2018-09-24-reach-router/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Routing with @reach/router in Gatsby v2
+date: 2018-09-24
+author: Amberley Romo
+tags: ["routing", "v2", "accessibility"]
+---
+
+We recently announced the [second major release of Gatsby](/blog/2018-09-17-gatsby-v2/) 🚀. One big v2 shift we want to highlight is the switch to using [@reach/router](https://reach.tech/router) to improve the accessibility of routing in Gatsby sites. What is @reach/router, and what are the benefits of undertaking the switch?
+
+## What is @reach/router?
+
+@reach/router is a successive routing library for React written and maintained by the founder of react-router, [Ryan Florence](https://twitter.com/ryanflorence). After [investigating the tradeoffs](https://github.com/gatsbyjs/gatsby/issues/5656), we made the leap to depending on @reach/router:
```

Comment:
"successive routing library" is a bit confusing to read as it's not obvious this is referring to react-router and it kinda looks like "successful" at first glance